### DISCUSSION TITLE
No data for 2024 Monthly point in API

### DISCUSSION THREAD

**Discussion by NessRo** [meteostat/meteostat-python#157] [General] [answered]
API is not returning values for 2024 monthly point data.

we use your API (paid) through RapidAPI and we are not getting 2024 data.

![code test - meteostat](https://github.com/meteostat/meteostat-python/assets/88949265/68196222-a540-498d-a650-eded6c4d27a1)

![code test - meteostat - results](https://github.com/meteostat/meteostat-python/assets/88949265/7679d3fd-4978-4ab1-ba2d-ecb6547ad92b)

**Comment by clampr** [comment]
Hi @NessRo, I think the first screenshot (daily data request) doesn't match the second one (monthly data response). Can you provide the actual `params` for your monthly data request?

**Reply by NessRo** [reply]
@clampr Sorry not sure what happened but i re-ran using this:

const axios = require('axios');

const options = {
  method: 'GET',
  url: 'https://meteostat.p.rapidapi.com/point/monthly',
  params: {
    lat: '52.5244',
    lon: '13.4105',
    start: '2020-01-01',
    end: '2024-05-01',
    alt: '43'
  },
  headers: {
    'X-RapidAPI-Key': 'MY KEY',
    'X-RapidAPI-Host': 'meteostat.p.rapidapi.com'
  }
};

try {
	const response = await axios.request(options);
	console.log(response.data);
} catch (error) {
	console.error(error);
}


This is the result 


![image](https://github.com/meteostat/meteostat-python/assets/88949265/9f81872a-c93e-4f87-9f1d-e91946f9b904)


Weird part is that it stops in September.

![image](https://github.com/meteostat/meteostat-python/assets/88949265/a8df2c20-f601-4e8c-8832-ce62fa63df5f)

**Comment by clampr** [comment, marked as answer]
Thanks for reporting this bug @NessRo. It was a strange issue, but I think I fixed it. I do get monthly data up until Feb 2024 now. Should be fine, given the fact the aggregations always lag behind a couple of months.
